roles/restore/tasks/restore.yml

```diff
@@ -0,0 +1,10 @@
+---
+# Run the Setup to restore tower
+- name: "[Tower] Run the Ansible Tower Setup Program with restore option"
+  become: true
+  command: ./setup.sh -e 'restore_backup_file={{ restore_location | quote }}' -r
```

Comment:
I think this should have a default of 

```
restore_location: "{{ tower_working_location }}/tower-backup-latest.tar.gz"
```

or perhaps be even better than that and have:
```
restore_file: "tower-backup-latest.tar.gz"
restore_location: "{{ tower_working_location }}/{{ restore_file }}"
```

Reply:
Updated